### DISCUSSION TITLE
FirstVDS Network interface

### DISCUSSION THREAD

**Discussion by AlexChuprikovCRS** [mikopbx/Core#380] [Q&A] [answered]
Есть варианты установки в яндекс, маил и прочие вещи... Я решил попробовать поставить на VDS. 
Столкнулся с проблемой. Сетевой интерфейс у VDSки - ens3, если поставить через примонтированый ISO образ, то у Askozia сетевой интерфейс eth0, с каким-то странным IP адресом (192.168.0.2 такого вида)... Есть ли вариант выставить другой сетевой интерфейс или настроить сетевой интерфейс? Выставление IP адреса в ручную не решает проблемы - сетка не работает.

**Comment by AlexChuprikovCRS** [comment, marked as answer]
Два дня бился. Помогла инструкция 
https://wiki.mikopbx.ru/faq:hetzner-cloud

**Reply by boffart** [reply]
Все верно, с некоторыми облаками нужна ручная настройка сети. Тут универсальное сложно что либо придумать. 
Пока лучше всего работает в Yandex облаке. 

Кстати еще один вариант установки. - это docker, но в продакш используют единицы.
